README.md

```diff
@@ -77,7 +88,51 @@ Clears the entire cache.
 
 Returns the current number of items in the cache.
 
+---
+
+### ReadThroughPromiseCache
```

Comment:
Could benefit from a sentence or two (or even a link out to an authoritative article on what a readthrough cache is, presuming we've implemented it correctly) to explain what this is and why you'd use it.

Reply:
[f3ef4cb](https://github.com/ardriveapp/promise-cache/pull/2/commits/f3ef4cb80cdc5e49a6dda6d453986fd5364e9961)